src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BALL_NUMBER_MAX_VALUE = 100;
+
+    public Ball getRandomBall() {
+        Color color = Color.valueOf(new ColorSupplier().getRandomColor());
```

Comment:
Overloaded expression, please introduce some variables. Also creation of Random and ColorSupplier objects should in class level variable